dogs.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog - Dogs</title>
+    <link rel="stylesheet" type="text/css" href="./styles/style.css">
+    <script src="https://kit.fontawesome.com/49a0808614.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+      <header class="header">
+        <img src="./images/adoptadog-logo.svg" alt="sitting dog sillouette logo">
+        <h3>Seattle's premiere dog adoption agency</h3>
+        <i class="fas fa-shopping-cart"></i>
+        <button class="checkout" type="button">
+            Checkout
+        </button>
+      </header>
+      <nav class="bar">
+        <ul>
+            <li><a href="default.asp">Home</a></li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <section class="dogs-main-container">
+       <h2>Here are all of our Dogs!</h2>
+        <div class="dog-1">
```

Comment:
all of these dogs share the same styling and can use the same class for styling .dog-1, .dog-2, etc  can all be covered by a single class and one css selector will hit them all.